tools/datatypes.txt

```diff
@@ -0,0 +1,310 @@
+fates_history_ageclass_bin_edges 1
```

Comment:
I'm wondering about this dataset. It's really long and detailed and I wonder if it has to correspond to code or data that's encapsulated elsewhere? If so I wonder if there's a way to generate it? It also might be data that doesn't change often so it might not be an issue.

It would also be helpful to allow for comments to give more of an explanation of what's in the file.